test/check-ostree

```diff
@@ -247,7 +230,7 @@ class OstreeRestartCase(MachineCase):
 
         # Check update data
         b.wait_visible("{0} div.tree".format(get_list_item(1)))
-        b.wait_in_text("{0} .os".format(get_list_item(1)), name)
+        b.wait_in_text("{0} .os".format(get_list_item(1)), "fedora-coreos")
```

Comment:
ah, I didn't think of that usage here -- that repeats the OS name a lot here. But *shrug*, we can re-factorize it later on as well.